src/applications/sco/components/MainContent/Update/OtherResources.jsx

```diff
@@ -41,34 +41,34 @@ const OtherResources = () => {
           <ul>
             <LiSpanAndVaLink
               download
-              hrefText="Download Flight - Enrollment Manager Quick Start Guide (PDF, 4 pages)"
```

Comment:
We want to keep the "Download__" text for all PDF links

Reply:
Done